Index: 1~10단계/input/25093_새싹.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/1~10단계/input/25093_새싹.py b/1~10단계/input/25093_새싹.py
new file mode 100644
--- /dev/null	(date 1652107000723)
+++ b/1~10단계/input/25093_새싹.py	(date 1652107000723)
@@ -0,0 +1,11 @@
+print()         ,r'"7
+r`-_   ,'  ,/
+ \. ". L_r'
+   `~\/
+      |
+      |         ,r'"7
+r`-_   ,'  ,/
+ \. ". L_r'
+   `~\/
+      |
+      |
\ No newline at end of file
